initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -559,26 +559,24 @@ stop_component() {
 
     eval $stopcmd
 
+    unlock ${LOCKPATH}
+    removePid ${PIDPATH}
+
     RESULT=0
-    local counter=0
-    while [[ $RESULT -eq 0 && $counter -le 20 ]]; do
+    local waittime=30
```

Comment:
since dali relies on dalistop, we could check to see if that's running on startup, and if so fail for dali.  And a force stop on any component wouldn't be hard to code.